### PR TITLE
FIPS_ENABLED-hosted cluster

### PR DESCRIPTION
**Pull request by huangmingxia**
Add hypershift fips compliance CI test job - hosted cluster
User story: [HOSTEDCP-163](https://issues.redhat.com/browse/HOSTEDCP-163)
test case: [OCP-43554](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-43554)
QE process: [OCPQE-15708](https://issues.redhat.com/browse/OCPQE-15708)

### CODE REVIEW

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f14

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f14

**Comment by LiangquanLi930** [comment]
@huangmingxia Could you help to add `destructive` job?

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f14

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f14

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f28-destructive

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f28-destructive

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -680,6 +680,38 @@ tests:
...
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-efs
+- as: aws-ipi-ovn-hypershift-fips-guest-p1-f14
+  cron: 25 9 12,27 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and @hypershift-hosted
+        and not @fips'
```
Comment (LiangquanLi930) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:690]: should be `and @fips`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 2]
```diff
@@ -680,6 +680,38 @@ tests:
...
+    - chain: openshift-e2e-test-hypershift-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: aws-ipi-ovn-hypershift-fips-guest-p1-f28-destructive
+  cron: 21 19 12 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and @hypershift-hosted
+        and not @fips and @destructive'
```
Comment (LiangquanLi930) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:706]: Same

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f14

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f28-destructive

**Comment by huangmingxia** [comment]
@liangxia Would you mind help review this PR when you free? Much thanks.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 3]
```diff
@@ -680,6 +680,38 @@ tests:
...
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-efs
+- as: aws-ipi-ovn-hypershift-fips-guest-p1-f14
+  cron: 25 9 12,27 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and @hypershift-hosted
+        and @fips'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:690]: `and @fips` should not needed here

Comment (huangmingxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: @liangxia Done. Thanks.

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f28-destructive

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-p1-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 4]
```diff
@@ -680,6 +680,37 @@ tests:
...
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and @hypershift-hosted
+        and @destructive'
+      EXTRA_ARGS: --fips
+      FIPS_ENABLED: "true"
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;~Serial&;Disruptive
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:709]: @kuiwang02  Is `TEST_FILTERS` correct and needed here ?

**Comment by LiangquanLi930** [comment]
> @kuiwang02 Is TEST_FILTERS correct and needed here ?

I can't confirm it, waiting for kuiwang's reply

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 4 above)
Comment (kuiwang02) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:709]: @huangmingxia @liangxia 
it should be `~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;Serial;Disruptive`

Comment (huangmingxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: @kuiwang02 @liangxia @LiangquanLi930 Thanks you all.

**Review by LiangquanLi930 — APPROVED** [review]

**Pull request merged**